docs/accounts/fxa-client-ios.md

```diff
@@ -0,0 +1,126 @@
+---
+id: fxa-client-ios
+title: iOS SDK
+---
+
+The **Firefox Accounts iOS SDK** provides a way for iOS applications to do the following:
+
+* Fetch scoped keys to provide a key for end-to-end encryption.
+* Get access to Firefox Sync keys to fetch the sync data.
```

Comment:
I'm not sure it's useful for clients to understand the distinction between the Firefox Sync keys and any other keys - they're all basically fetched and used in the same way, right?  Perhaps we can say something like:

* Fetch the user's profile information to personalize the application.
* Obtain OAuth tokens that can be used to access the user's data in Mozilla-hosted services like Firefox Sync.
* Obtain any client-side encryption keys that are needed to access that data.